shopelectro/settings/base.py

```diff
@@ -501,3 +501,8 @@ def get_robots_content():
 
 PRODUCTS_ON_PAGE_PC = 48
 PRODUCTS_ON_PAGE_MOB = 12
+
+TG_BOT_TOKEN = os.environ.get('TG_BOT_TOKEN')
+TG_REPORT_ADDRESSEES = os.environ.get(
+    'TG_REPORT_ADDRESSEE', '@artemiy312,@duker33'
```

Comment:
```suggestion
    'TG_REPORT_ADDRESSEES', '@artemiy312,@duker33'
```

Reply:
let's plz create some channel "shopelectro_health" or "shopeletro_report" and send stats there